### PR TITLE
Update installation scripts to reference `main` branch instead of `ma…

### PR DESCRIPTION
**Pull request by samWson**
These installation scripts got missed when fixes were made because of the shift from `master` to `main` as the primary development branch.

### CODE REVIEW

**Comment by samWson** [comment]
Hold on merging this PR. I thought the installation scripts were not up to date.

At least two people have been having errors installing because the installation scripts on the Exercism website still look like this:
```smalltalk
Iceberg remoteTypeSelector: #httpsUrl.
Metacello new 
 baseline: 'Exercism'; 
 repository: 'github://exercism/pharo-smalltalk/releases/latest';
 load.
 
#ExercismManager asClass welcome.
```

The `repository` part of that script should read like this: ` repository: 'github://exercism/pharo-smalltalk:main/releases/latest'`.
And it does in the installation documentation: https://github.com/exercism/pharo-smalltalk/blob/main/docs/INSTALLATION.md

At a guess the current version of the Exercism website is not talking updates from the repository anymore due to the imminent release of Exercism V3.

**Comment by Bajger** [comment]
Hi Sam, I think it is because Exercism-Smalltalk is probably not released with updated version of MD from main (done by @edward ). I guess someone from original authors @macta can do this?

**Comment by samWson** [comment]
@Bajger I think it's just the installation script.

When people who have had this issue try again with the script as it is in the repository they don't have any problems installing.

**Comment by Bajger** [comment]
Installation script is ok (as seen in https://github.com/exercism/pharo-smalltalk/blob/main/docs/INSTALLATION.md), but Exercism Pharo website (https://exercism.io/tracks/pharo/installation) uses version from release of this md file, but not actual version from main. Therefore I think new release of this project will make the change propagated to Exercism website.

**Comment by Bajger** [comment]
See https://github.com/exercism/pharo-smalltalk/releases/  - last update was in 2019.

**Comment by SleeplessByte** [comment]
cc @iHiD 

Could be that the name change didn't sync up. I've tagged Jeremy to confirm. It's also possible that _even_ if you do a new release that it isn't synced, and he'll have to manually push it to unbreak the site for pharo users.

Is there a release planned to check if that would solve it?

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -130,4 +130,4 @@ ExercismHttpClient
 
 ## How to Upgrade Pharo Exercism
 
-From time to time we may need you to update the libraries in your Pharo Exercism image. Please follow [these instructions](https://github.com/exercism/pharo-smalltalk/blob/master/docs/UPGRADE.md)
\ No newline at end of file
+From time to time we may need you to update the libraries in your Pharo Exercism image. Please follow [these instructions](https://github.com/exercism/pharo-smalltalk/blob/master/docs/UPGRADE.md)
```
Comment (SleeplessByte) [inline docs/INSTALLATION.md:133]:
```suggestion
From time to time we may need you to update the libraries in your Pharo Exercism image. Please follow [these instructions](https://github.com/exercism/pharo-smalltalk/blob/main/docs/UPGRADE.md)
```

**Comment by samWson** [comment]
Doing a bit more investigation into this. @iHiD suggested making a `master` branch that mimics `main`. I'll give that a go since it's quick to do. Then I'll have a look into the releases. I don't really know how they fit into this repo and I haven't given them any attention until now.

For clarity here is the error:
error-installing-exercism

File: docs/INSTALLATION.md (hunk 1 above)
Comment (samWson) [reply docs/INSTALLATION.md:133]: Thanks.

**Comment by samWson** [comment]
I've just created a `master` branch which is identical to `main`. I've tested the installation script again (from the Exercism website) and it works on Windows. I also fetched an exercise and ran the tests. No problems.

This PR should still get merged because you found a reference to `master` @SleeplessByte.

I still don't know what the releases are for since I've never used them. They don't seem to be used at all by the installation script given that making a `master` branch fixed the issue. I presume they are a way to download a Pharo image that already has exercism installed.

@exercism/reviewers could I please get an approval for this PR?

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ and launch Pharo with the exercism tools by pasting the following "load script":
 ```smalltalk
 ./pharo-ui Pharo.image eval "
 Iceberg remoteTypeSelector: #httpsUrl.
-Metacello new
- baseline: 'Exercism';
+Metacello new 
+ baseline: 'Exercism';
```
Comment (kotp) [inline docs/INSTALLATION.md:25]: Trailing white space introduced.
```suggestion
Metacello new
 baseline: 'Exercism';
```

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -65,11 +65,11 @@ Finally, copy and paste the following snippet into the playground:
 
 ```smalltalk
 Iceberg remoteTypeSelector: #httpsUrl.
-Metacello new
- baseline: 'Exercism';
+Metacello new 
+ baseline: 'Exercism';
```
Comment (kotp) [inline docs/INSTALLATION.md:69]:
```suggestion
Metacello new
 baseline: 'Exercism';
```
Trailing white space introduced.

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -65,11 +65,11 @@ Finally, copy and paste the following snippet into the playground:
...
 ```smalltalk
 Iceberg remoteTypeSelector: #httpsUrl.
-Metacello new
- baseline: 'Exercism';
+Metacello new 
+ baseline: 'Exercism'; 
  repository: 'github://exercism/pharo-smalltalk:main/releases/latest';
  load.
-
+
```
Comment (kotp) [inline docs/INSTALLATION.md:72]:
```suggestion

```

Trailing white space introduced.

**Review by kotp — CHANGES_REQUESTED** [review]
Trailing white space should not be introduced.

**Review by SleeplessByte — APPROVED** [review]

**Review by kotp — APPROVED** [review]

**Pull request merged**